src/d3a/d3a_core/singletons.py

```diff
@@ -16,7 +16,7 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 from d3a.d3a_core.global_objects import ExternalConnectionGlobalStatistics
-from d3a.models.myco_matcher import MycoMatcher
+from d3a.models.myco_matcher.myco_matcher import MycoMatcher
```

Comment:
You could eliminate this additional route with adding an `__all__ = []` statement in the `myco_matcher.__init__` right?

Reply:
This does make sense,
However in the `MycoMatcher` I do import the Internal and External Matchers, so if I put the `MycoMatcher` in the __init__ besides these two I will have a circular imports issue

Reply:
I see. leave it as is then.